app/assets/javascripts/nested_related_items.js

```diff
@@ -13,12 +13,21 @@
         $(this).removeClass('open');
         _this.addToggleLink($(this));
       });
+
+      _this.addToggleAll();
+      _this.toggleAll();
     },
 
     listItems: function(list) {
       return list.find(this.options.itemSelector);
     },
 
+    addToggleAll: function() {
+      var title = $('dt[title="Contains"]');
```

Comment:
I get kinda nervous about using the `title` attr in the selector here.  It could have its `dt` value changed by a `displayLabel` and we would lose this behavior.  If we pass the `el` variable when we call this from the `init` function we might be able to traverse the `dt`/`dd` structure to find the `dt` instead of relying on the label value itself.

Reply:
✅  Makes sense. I switched over to finding the `<dt>` immediately preceding the list's parent `<dd>`.